_redirects

```diff
@@ -71,3 +71,4 @@ https://openzeppelin-docs.netlify.com/* https://docs.openzeppelin.com/:splat 301
 /contracts/3.x/api/drafts /contracts/3.x/drafts 301
 /contracts/3.x/api/lifecycle /contracts/3.x/api/utils 301
 /contracts/3.x/api/ownership /contracts/3.x/api/access 301
+/contracts/api/deploy-ready /contracts/3.x/api/presets 301
```

Comment:
An unversioned URL should redirect to another unversioned URL. Though I don't know if this redirect is what we want... I'm getting lost with so many redirects. Have we published any links to this URL?

```suggestion
/contracts/api/deploy-ready /contracts/api/presets 301
```

Reply:
Yes, this was a mistake on my part. Thanks!